Initial implmentation

swhite24 opened the pull request:
Includes:
- Initial implementation of requesting data from the Venn Places API
- Cocoapods project setup
- Non useful test example

Needs:
- Publishing to cocoapods
- Possible test extension
- Possibly refactor the lock down allowed query parameters

swhite24 committed:
Initial commit

swhite24 committed:
Update gitignore to ignore test Pods

swhite24 committed:
Initial VennPlaces implementation

swhite24 committed:
Update tests

swhite24 committed:
Update podspec

swhite24 committed:
Merge master

swhite24 committed:
Update to actually use api key

swhite24 committed:
Add some docs

swhite24 committed:
Formatting

swhite24 committed:
More formatting

swhite24 committed:
Add contributing section to README

timmyg commented:
can you do an example project that hits the api with the test key and displays some results through the emulator?

and document how to build it, etc. for a n00b contributor

swhite24 committed:
Rework companion test project to be an application

swhite24 committed:
Remove old test project

swhite24 committed:
Update api to deliver dictionary / err to callback

swhite24 committed:
Update some docs

swhite24 commented:
Updated the companion test project to be an actual application, loading the names of places delivered from the API into a table view.

Also updated README to reference the new workspace (`PlacesTest/VennPlaces.xcworkspace`).  Open this to get started with building / running in the emulator.

swhite24 committed:
Remove some debugging logs

timmyg commented:
I'll look at this sometime this week... then we'll get you you HackOTR prize :)
